h2o-py/h2o/utils/metaclass.py

```diff
@@ -201,6 +200,7 @@ def __call__(self, clz):
         clz._bc = self
         new_clz = None
 
+        @wraps(clz.__init__)
```

Comment:
small fix here to ensure that the init signature and doc appears correctly when printing help in ipython/jupyter: noticed this when fixing the GridSearch docstring.